packages/variants/src/variants.ts

```diff
@@ -59,6 +59,7 @@ export function getNeutralVariant(theme: IPartialTheme): ITheme {
 
   const partialSemantic: Partial<ISemanticColors> = {
     bodyBackground: p.neutralLighter,
+    bodyStandoutBackground: p.themeDarkAlt,
```

Comment:
this should be one step up from neutralLighter, which is neutralLight
also need this slot for getSoftVariant
if the variantBorder values are finalized, i'd add those here too